### ISSUE TITLE
Show how many unique anonymous users have accessed a sharing link

### ISSUE THREAD

**Issue by kentonv** [sandstorm-io/sandstorm#2800]
This would be useful for detecting if a link has been shared to an unexpectedly wide audience.

**Comment by kpreid** [comment]
With the caveat that “anonymous users” is as fuzzy as “page views”…

**Comment by kentonv** [comment]
@kpreid Right. I figure we'll attempt to store a cookie (or rather a localStorage token) to detect repeat users but of course there's no way to tell if it's the same person repeatedly opening in new incognito windows.

**Comment by LinuxLover9** [comment]
I also would like to see who accesses the shared link, besides the intended audience. Though as flawed as it might be, it will be an indication that  something could be amiss. It's better than being totally in the dark...
